### PR TITLE
{bio}[foss/2018b] parasail 2.4

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/parasail/parasail-2.4-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+source_urls = ['https://github.com/jeffdaily/parasail/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['9eec7bfca011c2b311e652dbcba3475d43a7c094ca6983ffc51ee53b2cbb0343']
+
+builddependencies = [('CMake', '3.11.4')]
+
+sanity_check_paths = {
+    'files': ['bin/parasail_aligner', 'bin/parasail_stats',
+              'lib/libparasail.%s' % SHLIB_EXT, 'include/parasail.h'
+              ],
```
Comment (smoors) [inline easybuild/easyconfigs/p/parasail/parasail-2.4-foss-2018b.eb]: @fizwit the closing `]` seems out of place?
maybe just add to previous line like this:
```
    'files': ['bin/parasail_aligner', 'bin/parasail_stats',
              'lib/libparasail.%s' % SHLIB_EXT, 'include/parasail.h'],
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3104.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/dbbd2cc280bc27c35e9f15a43f121d24 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6183983cccf4af42b8c04436a0ac3a16 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2035.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a611127d08b2c7af82ca00d9346146b0 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @fizwit!

**Pull request merged**